docs/src/tutorials/tutorial_1.md

```diff
@@ -2,6 +2,350 @@
 
 ## Introduction
 
-If you are reading this tutorial, I am going to assume this is the first time you are using `Javis` to create an animation. In that case, welcome to `Javis`! 😃 By following this tutorial, we are going to make you a director of your very own animations written in pure Julia! 🎬 🎥
+If you are reading this tutorial, I am going to assume this is the first time you are using `Javis` to create an animation. 
+In that case, welcome to `Javis`! 😃 
+By following this tutorial, we are going to make you a director of your very own animations written in pure Julia! 🎬 🎥
 
 If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](../index.html#Installation).
+
+## Learning Outcomes
+
+In this tutorial you'll learn:
+
+- What `Javis.jl` is.
+- How to make a basic animation using `Action` objects.
+- Move objects created by `Javis`.
+- Elements of `Luxor.jl`.
+
+By the end of this tutorial, you will have made the following animation:
+
+![](assets/dancing_circles.gif)
+
+With all that said, let's dive in to this tutorial! ✨
+
+## So... What Is `Javis.jl`?
+
+`Javis.jl` is an abstraction on top of powerful graphics tools to make animations and visualizations easy to make.
```

Comment:
I can't figure out a better way but somehow I thing there must be a way to say it without a double "make" :smile: